pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherSingleActiveConsumer.java

```diff
@@ -70,6 +71,7 @@ public PersistentDispatcherSingleActiveConsumer(ManagedCursor cursor, SubType su
         this.cursor = cursor;
         this.readBatchSize = MaxReadBatchSize;
         this.serviceConfig = topic.getBrokerService().pulsar().getConfiguration();
+        this.dispatchRateLimiter = new DispatchRateLimiter(topic, name);
```

Comment:
To avoid additional object per-consumer, when the rate limiter per-subscription is disabled, would it be easy to initialize to null?